Makefile

```diff
@@ -127,7 +127,7 @@ unit-tests: ## Runs unit tests
 
 .PHONY: generate-mocks
 generate-mocks: ## Generates mocks and other autogenerated types
-	$(GOENVVARS) go generate ./...
+	mockery
```

Comment:
Heads up:
I have added this change as part of mine PR as well (https://github.com/0xPolygon/agglayer/pull/79/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R129-R130)